corehq/apps/repeaters/repeater_generators.py

```diff
@@ -40,7 +40,7 @@ def get_payload(self, repeat_record, payload_doc):
     def get_headers(self):
         return {'Content-type': self.content_type}
 
-    def get_test_payload(self):
+    def get_test_payload(self, domain=None):
```

Comment:
I don't follow 'to be safe' but I thought it's not ideal to force callers to pass an argument that doesn't need to be get passed. (changed it anyway)